roles/openshift_node/defaults/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 os_firewall_allow:
-- service: OpenShift kubelet
+- service: Kubernetes kubelet
```

Comment:
@detiber I would guess that one way to handle that and to avoid race is that iptables manipulations for each of those processes should be handled by openshift-ansible and not by individual process. 

Leaving iptables manipulations to user/system admin might be error prone IMHO.